src/test/java/org/folio/edge/rtac/MainVerticleTest.java

```diff
@@ -47,7 +51,7 @@
   private static final String badApiKey = apiKey + "0000";
   private static final String unknownTenantApiKey = ApiKeyUtils.generateApiKey(10, "bogus", "diku");;
```

Comment:
please clean up this extra `;` while you're changing this file

Reply:
done